sass/_chatbox.scss

```diff
@@ -121,6 +121,20 @@
             width: var(--mobile-chat-width);
         }
 
+        #service_administration_description {
+            color: #777;
```

Comment:
To make theming easier we should try to avoid hardcoding colors and instead use CSS properties.

Looks to me like here you can use `color: var(--secondary-color);`